src/styles/partials/components/_site-header.scss

```diff
@@ -39,20 +39,34 @@
 }
 
 @media screen and (max-width: $header-breakpoint) {
+  #search-form {
+    display: none;
+    padding: 15px;
```

Comment:
Should use scss spacing variable`$default-padding` here for padding.

Reply:
good catch, thank you.

Reply:
this has been addressed.